Practicals/prac_06/car.py

```diff
@@ -32,4 +37,5 @@ def drive(self, distance):
         return distance
 
     def __str__(self):
+        """Change class output into readable string."""
```

Comment:
your docstring is good, I saw the solution is something like """Return a string representation of a Car object.""" maybe it is more straightforward.